src/components/Calculator.jsx

```diff
@@ -3,30 +3,96 @@
 import React from 'react';
 import './Calculator.css';
 
```

Comment:
- [ ] OPTIONAL
You have **disabled ESLint checks** on parts of your code (_line 1 of Calculator.jsx_). Please enable all ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust and clean code.